hub/api/tensor.py

```diff
@@ -43,6 +45,8 @@ def __init__(
 
         self.chunk_engine = ChunkEngine(self.key, self.storage)
 
+        self._sample: Optional[Tuple[int, int]] = None
```

Comment:
what is this? can you rename it to be more descriptive and if it's still not clear add a comment